src/hugr/validate.rs

```diff
@@ -808,7 +808,11 @@ mod test {
         parent: Node,
         predicate_size: usize,
     ) -> (Node, Node, Node, Node) {
-        let const_op = ops::Const::new(ConstValue::simple_predicate(0, predicate_size)).unwrap();
+        let const_op = ops::Const::new(
```

Comment:
not `ops::Const::simple_predicate(0, predicate_size)`? I guess that's the same thing, but let's test more prod code with less test code ;-)